docs/core/learn-more/authentication.md

```diff
@@ -1,35 +1,97 @@
+
+## Authorization
 
 There are two APIs in Chain Core: the **client API** and the **network API**.
 
-The client API is used by the SDKs and the dashboard to communicate with Chain Core. The network API is used by [network operators](blockchain-operators.md).
+The client API is used by the SDKs and the dashboard to communicate with Chain
+Core. The network API is used by [network operators](blockchain-operators.md).
 
-Each API is authenticated using access tokens with HTTP Basic Authentication. 
+There are four policies available to grant an individual authentication method
+access to one or both APIs:
 
-For convenience, **when accessing from localhost, neither API requires authentication**.
+* **client-readwrite**: Full access to the Client API.
+* **client-readonly**: Access to read-only Client endpoints. This is a strict
+subset of the `client-readwrite` policy.
+* **monitoring**: Access to monitoring-specific endpoints. This is a strict
+subset of the `client-readonly` policy.
+* **network**: Access to the Network API.
 
-## Creating access tokens
+## Setting Up
 
-_The instructions in this section require having the Go programming environment installed and the `PATH` variable correctly configured. See [the Chain Core Readme file](https://github.com/chain/chain/blob/main/Readme.md) for details._
+### Mac, Windows and Docker
 
-Both client and network access tokens are created in the dashboard. However, when deploying Chain Core to a non-local environment, you will not be able to access the dashboard, because you will not yet have a client access token. Therefore, you must use the `corectl` command line tool to create your first client access token. After that, you can use that access token to login to the dashboard and create additional access tokens.
+Chain Core Developer Edition is distributed with a flag automatically permitting
+all requests from localhost without configuring an authorization method.
 
-Install the `corectl` command line tool:
+If you are accessing a Chain Core DE server from an external URL, you can
+follow the instructions below. Otherwise, skip ahead to
+[granting access](#granting-access).
 
-```bash
-go install ./cmd/corectl
-```
+### Remote Server
 
-Create a **client access token** using `corectl`:
+When deploying Chain Core to a non-local environment, you will not be able to
+access the Dashboard or APIs to create authorization grants. Therefore, you
+must use the `corectl` command line tool to create your first authorization
+grant. After that, you can use that token or certificate to create additional
+authorizations via the Dashboard or SDKs.
 
-```bash
-corectl create-token <name>
+[sidenote]
+
+Before proceeding, make sure you have `corectl` installed on your system. If
+it's not already present, see
+[installing `corectl`](../reference/corectl#installation).
+
+[/sidenote]
+
+Create a new **access token** and give it the **client-readwrite** policy:
+
+```
+corectl create-token <name> client-readwrite
```

Comment:
Good question. I think I proposed that as a strawman for a shorter syntax. Is there a current corectl syntax for adding an authorization grant? I didn't see one.